src/EventManagement.Infrastructure/ApplicationDbContext.cs

```diff
@@ -1,37 +1,37 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore;
-using losol.EventManagement.Models;
-
-namespace losol.EventManagement.Data
-{
-    public class ApplicationDbContext : IdentityDbContext<ApplicationUser>
-    {
-        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
-            : base(options)
-        {
-        }
-
-
-
-
-        protected override void OnModelCreating(ModelBuilder builder)
-        {
-            base.OnModelCreating(builder);
-            // Customize the ASP.NET Identity model and override the defaults if needed.
-            // For example, you can rename the ASP.NET Identity table names and more.
-            // Add your customizations after calling base.OnModelCreating(builder);
-
-        }
-
-        public DbSet<losol.EventManagement.Data.ApplicationUser> ApplicationUsers { get; set; }
-        public DbSet<losol.EventManagement.Models.EventInfo> EventInfos { get; set; }
-        public DbSet<losol.EventManagement.Models.Registration> Registrations { get; set; }
-        public DbSet<losol.EventManagement.Models.PaymentMethod> PaymentMethods { get; set; }
-        public DbSet<losol.EventManagement.Models.Product> Products { get; set; }
-        public DbSet<losol.EventManagement.Models.ProductVariant> ProductVariant { get; set; }
-    }
-}
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using losol.EventManagement.Domain;
+
+namespace losol.EventManagement.Infrastructure
+{
+    public class ApplicationDbContext : IdentityDbContext<ApplicationUser>
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
+            : base(options)
+        {
+        }
+
+
+
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+            // Customize the ASP.NET Identity model and override the defaults if needed.
+            // For example, you can rename the ASP.NET Identity table names and more.
+            // Add your customizations after calling base.OnModelCreating(builder);
+
+        }
+
+        public DbSet<ApplicationUser> ApplicationUsers { get; set; }
+        public DbSet<EventInfo> EventInfos { get; set; }
+        public DbSet<Registration> Registrations { get; set; }
+        public DbSet<PaymentMethod> PaymentMethods { get; set; }
+        public DbSet<Product> Products { get; set; }
+        public DbSet<ProductVariant> ProductVariant { get; set; }
```

Comment:
This is my mistake, but it should be ProductVariants. We´ll take that in the next iteration..

Reply:
Yes, let's change it after this PR is merged.